propolis/src/hw/virtio/pci.rs

```diff
@@ -186,10 +185,10 @@ impl PciVirtioState {
         cfg_sz: usize,
     ) -> (Self, pci::DeviceState) {
         let mut builder = pci::Builder::new(pci::Ident {
-            vendor_id: VIRTIO_VENDOR,
+            vendor_id: VENDOR_VIRTIO,
             device_id: dev_id,
-            sub_vendor_id: VIRTIO_VENDOR,
-            sub_device_id: dev_id - 0xfff,
+            sub_vendor_id: VENDOR_OXIDE,
+            sub_device_id: super::bits::virtio_sub_dev_id(dev_id),
```

Comment:
These could probably just be passed through as parameters from the specific device?  We could then make them configurable later, if it was required for something like testing.

Reply:
Fair enough, but we don't have any such use cases today? What cases do you see it being useful to configure the IDs in a test?